### ISSUE TITLE
Cannot start marathon (1.4.0-RC3) with `default_accepted_resource_roles = *`

### ISSUE THREAD

**Issue by rhruiz** [mesosphere/marathon#4854]
When starting marathon 1.4.0-RC3 (from the packaged version in http://repos.mesosphere.com/debian/pool/main/m/marathon/marathon_1.4.0-1.0.562.debian81_amd64.deb) and by using a file `/etc/marathon/conf/default_accepted_resource_roles` with the content `*` marathon crashes on startup with the message on `journalctl`:

```
Dec 14 20:40:55 cpro33659 marathon[4089]: Exception: java.lang.NullPointerException thrown from the UncaughtExceptionHandler in thread "main"
Dec 14 20:40:55 cpro33659 systemd[1]: marathon.service: main process exited, code=exited, status=1/FAILURE
Dec 14 20:40:55 cpro33659 systemd[1]: Unit marathon.service entered failed state.
```

**Comment by jdef** [comment]
related to
https://github.com/mesosphere/marathon/commit/85a32314d6c60d620c1fa2cc8d5660b23190cf47#diff-3903349d17af1178a3a2979b01a823dc
?

**Comment by unterstein** [comment]
Nope, 85a32314d6c60d620c1fa2cc8d5660b23190cf47#diff-3903349d17af1178a3a2979b01a823dc is in RC3

**Comment by rhruiz** [comment]
@jdef @unterstein the `/usr/bin/marathon` in package contains contains the lines in the diff:

```
       clean_cmd+=( "${i}" )
     fi
   done
   cmd_to_env "${clean_cmd[@]}"
   run_jar
 else
   cmd_to_env "${cmd[@]}"
   logged marathon run_jar "$@"
 fi
```

**Comment by unterstein** [comment]
Can you post the command which tries to start marathon?

**Comment by rhruiz** [comment]
What shows up in ps is:

`root     21738 57.0  4.8 2441212 98672 ?       Ssl  11:31   0:07 java -Djava.library.path=/usr/local/lib:/usr/lib:/usr/lib64 -Djava.util.logging.SimpleFormatter.format=%2$s%5$s%6$s%n -Xmx512m -cp /tmp/marathon-f57d92a5a7042cb76d343f637a2f99dd601e2e31.jar mesosphere.marathon.Main
`

**Comment by unterstein** [comment]
If you remove the config file, does Marathon startsie expected?

**Comment by rhruiz** [comment]
yes, it does start normally.

And if I set if as an environment variable, I get this error:

`Dec 16 16:47:37 cpro33659 marathon[16585]: /usr/bin/marathon: line 66: export: `MARATHON_CMD_INITRD.IMG=lib': not a valid identifier`

**Comment by unterstein** [comment]
Does this happen with other configurations as well or just with this one?

**Comment by rhruiz** [comment]
it seems to be related to the `*`, all other configurations are directories or files, they are working fine.

**Comment by rhruiz** [comment]
just tested with other value other than `*` and it fails with the same java.lang.NullPointerException

**Comment by unterstein** [comment]
Thanks @rhruiz for reporting!

We fixed this within #4890 which should land soon and will be included in the next RC.

**Comment by unterstein** [comment]
Just merged #4890 which solves this issue :-)
